client/app/lib/providers/computecontroller.coffee

```diff
@@ -1258,6 +1301,33 @@ module.exports = class ComputeController extends KDController
     else showError 'Failed to share credential'
 
 
+  removeClonedFromAttr: (stackTemplate, callback = kd.noop) ->
```

Comment:
yes, we may need to move some functionality into backend like these. we'll need such functionality over api support and in that case we'll end-up with duplicate codes.